app/pages/health/routes.test.js

```diff
@@ -0,0 +1,68 @@
+import { status } from './status';
+import { testHealthStatus } from '../../test-utils/testFunctions/testHealthStatus';
+
+jest.mock('../../apiProvider');
+
+describe('GET /health', () => {
+  describe('GET /live', () => {
+    it('should return \'Healthy\' when app is running', testHealthStatus('/live', {
+      expected: status.healthy,
+    }));
+  });
+
+  describe('GET /ready', () => {
+    it('should return \'Healthy\' when BAPI and DAPI are both \'Healthy\'', testHealthStatus('/ready', {
```

Comment:
please change to 
testHealthStatus(
  route: '/ready',
  BAPI: status.healthy,
  DAPI: status.healthy,
  expected: status.healthy,
)
 for each test (and you will have to update the `testHealthStatus` function to accept the params this way)